.github/optimize/scripts/build-docker-image.sh

```diff
@@ -18,7 +18,7 @@ docker buildx create --use
 export VERSION="${VERSION}"
 export DATE="$(date +%FT%TZ)"
 export REVISION="${REVISION}"
-export BASE_IMAGE=docker.io/library/alpine:3.22.0
+export BASE_IMAGE=docker.io/library/alpine:3.23.0
```

Comment:
Let's test if reverting this fixes CI
```suggestion
export BASE_IMAGE=docker.io/library/alpine:3.22.0
```